lib/zmq-kernel.js

```diff
@@ -168,6 +172,7 @@ export default class ZMQKernel extends Kernel {
 
   restart(onRestarted: ?Function) {
```

Comment:
In this `restart` entry point, we need to prevent simultaneous restart. like this
```javascript
    if (this.executionState === "restarting") {
      return
    }
```

since finish timing is monitor called `done`. but it happens async.